plugins/services/src/js/components/VolumeTable.js

```diff
@@ -130,25 +130,21 @@ class VolumeTable extends React.Component {
 
   renderIDColumn(prop, row) {
     let id = row[prop];
-    let params = {volumeID: global.encodeURIComponent(id)};
-    let routes = this.context.router.getCurrentRoutes();
-    let currentRoutePath = routes[routes.length - 1].path;
+    let {nodeID, taskID} = this.props.params;
+    let volumeID = global.encodeURIComponent(id);
+    let routes = this.props.routes;
+    let currentroutePath = routes[routes.length - 1].path;
     let routePath = null;
 
-    if (currentRoutePath === '/services/overview/:id') {
-      routePath = '/services/overview/:id/volumes/:volumeID';
-      params.id = this.props.params.id;
-    } else if (currentRoutePath === '/services/overview/:id/tasks/:taskID/volumes') {
-      routePath = '/services/overview/:id/tasks/:taskID/volumes/:volumeID';
-      params.id = this.props.params.id;
-      params.taskID = this.props.params.taskID;
-    } else if (currentRoutePath === '/nodes/:nodeID/tasks/:taskID/volumes') {
-      routePath = '/nodes/:nodeID/tasks/:taskID/volumes/:volumeID';
-      params.nodeID = this.props.params.nodeID;
-      params.taskID = this.props.params.taskID;
+    if (currentroutePath === '/services/overview/:id') {
+      routePath = `/services/overview/${this.props.params.id}/volumes/${volumeID}`;
+    } else if (currentroutePath === '/services/overview/:id/tasks/:taskID/volumes') {
+      routePath = `/services/overview/${this.props.params.id}/tasks/${taskID}/volumes/${volumeID}`;
```

Comment:
In this particular case I'd `[].join('/')` them instead. Breaking a route template string into multiple lines just to go under 80 chars doesn't feel good to me.